doc/changelog.md

```diff
@@ -71,7 +71,7 @@
   [(#725)](https://github.com/PennyLaneAI/catalyst/pull/725)
 
   Although library code is not meant to be targeted by Autograph conversion,
-  it sometimes make sense to enable it for specific submodules that might 
```

Comment:
I just noticed now that my editor trimmed some trailing whitespace. Should I leave this change, or add the whitespace back in?

Reply:
You can leave it like this, normally all files should be whitespace trimmed and final-newlined :)